README.md

```diff
@@ -1,13 +1,13 @@
 # 'Govee' integration
 
-The Govee integration allows you to control and monitor lights using the Govee API.
+The Govee integration allows you to control and monitor lights and switches using the Govee API.
 
 ## Installation
 
-* The installation is done inside [HACS](https://hacs.xyz/). If you don't have HACS, we recommend you install it first.
-* Once HACS is installed just search for the 'Govee' integration there, and click Install.
-* Then click Configuration in the sidebar and then click Integrations. Click the + icon to add "Govee" to your Home Assistant installation. An API key
-is required, you need to obtain it in the 'Govee Home' app on your mobile, in the User menu - About us - Apply for API Key. The key will be sent to you by email.
+* The installation is done inside [HACS](https://hacs.xyz/) (Home Assistant Community Store). If you don't have HACS, you must you install it before adding this integration. [Installation instructions here.](https://hacs.xyz/docs/setup/download)
```

Comment:
`If you don't have HACS, you must you install it before adding this integration. [Installation instructions here.](https://hacs.xyz/docs/setup/download)`
Should have the second you removed for "...you must you..."

Reply:
2nd grade English abilities was not a qualification for creating a Github account. :) 
Alrighty, I'll see if I can fix that.

Reply:
and switches 

- would be nice, but are we supporting switches? Not that I know of.
thanks for the other changes. Will merge them once we know if somehow any switch works.